R/ts_plot.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' Convert csv file into a time series object, decompose it into trend, seasonality/cyclicity
 #' and noise/remainder/error and plot the raw data and the decomposed components.
-#' If the time series can't be decomposed, the function will just return the line chart of the raw data. 
+#' If the time series can't be decomposed, the function will just return the line chart of the raw data.
 #'
 #' @param data str: the path of the csv file
 #' @param col str: the column to be analyzed
```

Comment:
```suggestion
#' @param data dataframe: a data.frame or tibble
#' @param col str: the column to be analyzed
```
This function should take in  either a data.frame or a tibble. It introduces needless complexity to take in a `.csv` file. it is also inconsistent with the rest of the package functions which take in dataframes.